This is not how you enter Cornell Tech faculty - please see one of your colleague's listings.

```
Sanity checking the commit. Please check any issues raised here.
Processing Nikhil Garg 0001,Cornell Tech,https://gargnikhil.com/,[8](https://github.com/emeryberger/CSrankings/actions/runs/6841512312/job/18601948608?pr=6199#step:6:9)qSK3noAAAAJ
  ERROR: This institution (Cornell Tech) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3ANikhil%20Garg%200001:$%3A&format=json&c=10
  Checking homepage URL (https://gargnikhil.com/)
***Test failure for Nikhil Garg 0001***.
Error: Process completed with exit code 255.
```

--- a/csrankings-n.csv
+++ b/csrankings-n.csv
@@ -480,7 +480,7 @@ Nihar Bhadresh Shah,Carnegie Mellon University,http://www.cs.cmu.edu/~nihars,BF3
 Nihat Altiparmak,University of Louisville,http://cecs.louisville.edu/nihat,HvPsovYAAAAJ
 Nika Haghtalab,Univ. of California - Berkeley,https://people.eecs.berkeley.edu/~nika/,C6pnolkAAAAJ
 Nikhil Bansal 0001,University of Michigan,https://www.win.tue.nl/~nikhil,3Lmd6AMAAAAJ
-Nikhil Garg 0001,Cornell Tech,https://gargnikhil.com/,8qSK3noAAAAJ
+Nikhil Garg 0001 [Tech],Cornell University,https://gargnikhil.com/,8qSK3noAAAAJ
 Nikhil Ranjan Pal,ISI Kolkata,https://www.isical.ac.in/~nikhil,IGIvYfAAAAAJ
 Niki Kittur,Carnegie Mellon University,https://www.hcii.cmu.edu/people/aniket-kittur,2vZ5zRQAAAAJ
 Niki Pissinou,Florida International University,https://www.cis.fiu.edu/faculty-staff/pissinou-niki,NOSCHOLARPAGE

